src/javascript/app_2/pages/trading/components/elements/header.jsx

```diff
@@ -1,35 +1,63 @@
 import React from 'react';
-import { ToggleDrawer, DrawerItem } from './drawer.jsx';
+import { ToggleDrawer, MenuDrawer, DrawerItem } from './drawer.jsx';
+import AccountSwitcher from './account_switcher.jsx';
+import Button from '../form/button.jsx';
 import Url from '../../../../../_common/url';
+import { localize } from '../../../../../_common/localize';
 
 class TradingHeader extends React.Component {
+
     render() {
+        const menu_link_is_active = (name) => {
+            const pathname = window.location.pathname;
+            if (pathname.indexOf(name.toLowerCase()) >= 0) {
+                return true;
+            }
+            return false;
```

Comment:
same here, but it's up to you :)
`const menu_link_is_active = (name) => (window.location.pathname.indexOf(name.toLowerCase() >= 0); `